gcp/README.md

```diff
@@ -15,42 +15,44 @@ You can now authenticate a GCP provider in Stacks with the following setup:
 ```hcl
 # main.tfdeploy.hcl
 
-store "varset" "credentials" {
-  id       = <output.varset from earlier Terraform execution>
-  category = "terraform"  
+identity_token "jwt" {
+  audience = ["hcp.workload.identity"]
 }
 
-deployment "development" {
+deployment "staging" {
   inputs = {
-    google_project               = <the same project used in the setup>
-    google_region                = <the same region used in the setup>
-    google_credentials           = store.varset.credentials.gcp_credentials
-  }  
+    jwt = identity_token.jwt.jwt
```

Comment:
Can we give some of these variables more descriptive names? I think it'd be really helpful for folks to know how these JWT tokens are commonly used and having lines like `jwt = identity_token.jwt.jwt` is a little hard to parse.

Reply:
i went ahead and updated it to be `identity_token "gcp" {}` since we see a similar approach to [describing the identity_token here with the use of aws](https://github.com/hashicorp-guides/terraform-stacks-identity-tokens/pull/6/commits/13771f6c1e66a15240637ad4358f20862dfd5775)

Reply:
Thank you! This looks great.